pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderToken.java

```diff
@@ -202,6 +211,10 @@ private static String validateToken(final String token) throws AuthenticationExc
 
             return jwt;
         } catch (JwtException e) {
+            if (e instanceof ExpiredJwtException) {
+                String expiredTime = new Date(1000L * (Integer) ((ExpiredJwtException) e).getClaims().get("exp")).toString();
+                expiredTokenMetrics.labels(expiredTime).inc();
```

Comment:
This will create an unbound cardinality in the labels values. What's the rational to include the absolute time in the metric?

Reply:
Thanks. It seems unnecessary to record expired time here. Here I made some changes: Remove the `expiredTime` label in `expiredTokenMetrics`, use `Histogram` to record tokens that are about to expire and the corresponding expiration time.